.spectral.yaml

```diff
@@ -1 +1,35 @@
 extends: 'spectral:oas'
+
+aliases:
+  SchemaResponsePath:
+    - "$.paths[?(@.ref !== './resources/ApplicationAdvancedSettings.yaml')].*.responses.*.content[*].schema"
+  SchemaResponsePropertyObject:
+    - "#SchemaResponsePath.properties"
+  SchemaRequestPath:
+    - "$.paths[?(@.$ref !== './resources/ApplicationAdvancedSettings.yaml')].*.requestBody.content[*].schema"
```

Comment:
so I have tried here to add an exception for the advanced settings file but it does not work. I have tried many combinations but I couldn't find a way to make it work. I have validated this JSONPAth with the tool suggested by spectral (http://jsonpath.com/) and it is correct but spectral is still not taking it into account.

I'm abandoning this since it took me too much time already. let me know if you find any solution for this @benamib , @florianlepont